src/scales/scale.time.js

```diff
@@ -37,10 +41,107 @@ module.exports = function(Chart) {
 			},
 		},
 		ticks: {
-			autoSkip: false
+			autoSkip: false,
+			mode: 'linear',   // 'linear|series'
+			source: 'auto'    // 'auto|labels'
 		}
 	};
 
+	function sorter(a, b) {
+		return a - b;
+	}
+
+	/**
+	 * Returns an array of {time, pos} objects used to interpolate a specific `time` or position
```

Comment:
just one minor request on the doc here. is the position you're referring to here the position in pixels?